### ISSUE TITLE
Lambda Node 4.3 Depricated 

### ISSUE THREAD

**Issue by jontonsoup** [convox/rack#2661]
I woke up to this email from amazon. Moving forward, I think it makes sense to make new lambda functions on a more recent version of node and have an upgrade path for the old ones. 

Thanks!

```
Your AWS Account currently has one or more Lambda functions configured to use the Node.js v4.3 runtime. The Node Foundation declared End-of-Life (EOL) for Node.js v4 on April 30, 2018. As a result, this version of Node.js is no longer receiving bug fixes, security updates, or performance improvements from the Node Foundation. Per the AWS Lambda runtime support policy [2] , language runtimes that have been end-of-lifed by the supplier are deprecated in AWS Lambda. While invocations of Lambda functions configured to use Node.js v4.3 will continue to work normally, the ability to create new Lambda functions configured to use the Node.js v4.3 runtime will be disabled on July 31, 2018. Code updates to existing functions using Node.js v4.3 will be disabled on October 31, 2018. We strongly encourage you to update all your functions to a newer version of Node.js (v6.10 or v8.10) so that you continue to benefit from important security, performance, and functionality enhancements offered by the Node Foundation via more recent releases. 
```

**Comment by mwarkentin** [comment]
We got the same email. We have 43 convox lambda functions on node 4.3 in our production environment - probably 200+ across all of our racks.

**Comment by bladealslayer** [comment]
@ddollar I see that #2669 was merged and released, but a fully updated rack (`20180705140223`) seems to still be creating `syslog` resource functions with 4.3 as runtime. Any ideas why that may be?
